wp-content/themes/redmond/web/scss/layout/_footer.scss

```diff
@@ -1,3 +1,56 @@
 .footer {
-
+  background-color: $secondary-color;
+  padding: rem-calc(36px) rem-calc(80px);
+  color: $primary-light-color;
+  h2 {
+    color: $primary-light-color;
+    font-family: 'Lato', sans-serif;
+    font-size: rem-calc(19px);
+    font-weight: normal;
+    & small {
+      color: $primary-light-color;
+      font-style: normal;
+      text-transform: uppercase;
+      font-size: rem-calc(16px);
+    }
+  }
+  .footer-icons {
+    border-bottom: 1px #b8bfc2 solid;
+    margin-bottom: rem-calc(20px);
+    @include breakpoint(large) {
+      padding: 0 rem-calc(75px);
+    }
+    img.footer-icon {
+      margin-top: rem-calc(12px);
+    }
+  }
+  .footer-content {
+    .icon-title {
+      margin-bottom: rem-calc(15px);
+    }
+    img.logo {
+      width: 100%;
+      max-width: 250px;
+      margin-bottom: rem-calc(28px);
+    }
+  }
+  .facebook, .twitter, .pinterest, .google_plus {
+    width: rem-calc(28px);
+    height: rem-calc(28px);
+    border-radius: rem-calc(14px);
+    background-color: #b8bfc2;
```

Comment:
make this a var in settings. call it footer-social-background. add a hover effect that scales the color to lighten by 15%. there are some examples of scaling if you look at the button hover background color defined in settings.